apps/web/src/App/Router/AppRouter.tsx

```diff
@@ -114,10 +114,17 @@ function AppRouter(): ReactElement {
                           path="/airdrop"
                           element={<Airdrop></Airdrop>}
                         ></Route>
-                        {/*<Route
-                          path="/staking"
-                          element={<Staking></Staking>}
-                          ></Route>*/}
+                        {
+                          <Route
+                            path="/staking"
```

Comment:
Node staking or Staking Program?

I think the paths should reflect the text on the main menu

Reply:
staking-program